share/completions/blender.fish

```diff
@@ -0,0 +1,125 @@
+function __blender_list_scenes -a file
+    stdbuf --output=0 blender --background test.blend --python-expr 'import bpy
+
+for name in [scene.name for scene in list(bpy.data.scenes)]:
+    print(name)' | head --lines=-2 | tail --lines=+6
+end
+
+function __blender_list_addons
+    ls --format single-column /usr/share/blender/scripts/addons --color=never |
```

Comment:
See https://github.com/fish-shell/fish-shell/pull/9903#discussion_r1271431744

Reply:
(this is why splitting these up into separate PRs is awkward - you're adding code of the same form to all of these, but now I need to cross-reference my comments from the other PRs)